client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -136,6 +137,66 @@ public void testGet() {
         getAndExistsTest(Request::get, "GET");
     }
 
+    public void testMultiGet() throws IOException {
+        Map<String, String> expectedParams = new HashMap<>();
+        MultiGetRequest multiGetRequest = new MultiGetRequest();
+        if (randomBoolean()) {
+            String preference = randomAlphaOfLength(4);
+            multiGetRequest.preference(preference);
+            expectedParams.put("preference", preference);
+        }
+        if (randomBoolean()) {
+            multiGetRequest.realtime(randomBoolean());
+            if (multiGetRequest.realtime() == false) {
+                expectedParams.put("realtime", "false");
+            }
+        }
+        if (randomBoolean()) {
+            multiGetRequest.refresh(randomBoolean());
+            if (multiGetRequest.refresh()) {
+                expectedParams.put("refresh", "true");
+            }
+        }
+
+        int numberOfRequests = randomIntBetween(0, 32);
```

Comment:
what happens when we add no items?

Reply:
In this test it is fine, but on the server side we do fail if no items have been specified (see `MultiGetRequest#validate(...)`).